### PR TITLE
Creating behavioral protocol 

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: bids2openminds/converter.py [hunk 1]
```diff
@@ -31,11 +29,14 @@ def convert(input_path, output_path=None, multiple_files=False, include_empty_pr
     [subjects_dict, subject_state_dict, subjects_list] = main.create_subjects(
         subjects_id, layout_df, bids_layout, collection)
 
+    behavioral_protocols, behavioral_protocols_dict = main.create_behavioral_protocol(
```
Comment (lzehl) [inline bids2openminds/converter.py:32]: I don't understand the double output. Why do you need that?

File: test/test_example_datasets_click.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from bids2openminds.converter import convert_click
 from click.testing import CliRunner
 
-(test_data_set, number_of_openminds_files) = ("ds003", 100)
+(test_data_set, number_of_openminds_files) = ("ds003", 101)
```
Comment (lzehl) [inline test/test_example_datasets_click.py:6]: there should be a test that checks for the number of tasks (at least)

**Review by lzehl — CHANGES_REQUESTED** [review]
maybe this just requires clarification. please check.

File: bids2openminds/converter.py (hunk 1 above)
Comment (lzehl) [reply bids2openminds/converter.py:32]: note: only create dict and create list from that on demand

Comment (lzehl) [reply bids2openminds/converter.py:32]: dont change this now, raise issue for a code cleanup for the second release

File: test/test_example_datasets_click.py (hunk 2 above)
Comment (lzehl) [reply test/test_example_datasets_click.py:6]: leave it for later if this takes too much time now

**Review by apdavison — CHANGES_REQUESTED** [review]
`test_bids_examples.py` needs to be updated

**Pull request merged**